docs/root/_static/css/envoy.css

```diff
@@ -14,3 +14,12 @@ table.docutils div.line-block {
   overflow-wrap: break-word;
   max-width: 1000px;
 }
+
+/* To style the API version label of a search result item */
+.api-version-label {
+  border-radius: 20%;
+  background-color: #c0c0c0;
```

Comment:
"silver". We can always change it later when we want to re-style the entire docs.

<img width="1259" alt="image" src="https://user-images.githubusercontent.com/73152/92980955-26cca500-f4c2-11ea-8a12-494f5ff786c7.png">